test/integration/local/conftest.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption('--docker-base-name', default='sagemaker-tensorflow-serving')
+    parser.addoption('--framework-version', default='1.12', choices=['1.11', '1.12'])
```

Comment:
1. could you please add `--tag`, `--accelerator-type` also as an argument?
`--tag` will allow using any custom tagging scheme user intends to

2. Do the tests depend on AWS region? If so, that will be a nice argument to have

3. Please update the README with the features added in this PR 

4. Can we remove the `choices` and `defaults` for framework-version so that this file doesn't need to be modified for each release? Maybe have `required=True`

Reply:
--accelerator-type is an existing option in ei test's fixuture https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/test/integration/sagemaker/test_ei.py#L50

local test does not use AWS resources

README updated